### PR TITLE
Added standard orthography to Estonian

### PR DESCRIPTION
**Pull request by Bibiko**

### CODE REVIEW

File: cldf/requirements.txt [hunk 1]
```diff
@@ -1,48 +1,55 @@
...
+configparser==5.0.2
 csvw==1.11.0
 decorator==4.4.2
+fuzzywuzzy==0.18.0
 gitdb==4.0.7
 greenlet==1.1.0
+html5lib==1.1
 idna==3.2
-idspy==0.2.0
+-e git+https://github.com/intercontinental-dictionary-series/idspy.git@d2a6738c1388a31f4f427b0fef1d980e5ce65ce1#egg=idspy
```
Comment (chrzyki) [inline cldf/requirements.txt]: Would an `idspy` release maybe good before `v4.2`?

**Review by chrzyki — APPROVED** [review]

File: cldf/requirements.txt (hunk 1 above)
Comment (Bibiko) [reply cldf/requirements.txt]: Definitely yes!

Comment (Bibiko) [reply cldf/requirements.txt]: I'd go with idspy version 0.3.0

Comment (chrzyki) [reply cldf/requirements.txt]: Cool, thanks. Yes, `0.3.0` sounds good.

**Pull request merged**